Makefile

```
@@ -0,0 +1,144 @@
+	@echo "  save            - Save image to tar ($(IMAGE_TAR))"
+	@echo "  load            - Load image into containerd (sudo ctr -n $(CTR_NS) images import)"
+	@echo "  k8s-namespace   - Create namespace ($(NAMESPACE))"
+	@echo "  k8s-config      - Create/Update ConfigMap (non-secrets)"
+	@echo "  k8s-secrets     - Create/Update Secret from env (DJANGO_SECRET_KEY, DATABASE_URL or DB_*)"
+	@echo "  deploy          - Apply PVC, Deployment, Service, Ingress"
+	@echo "  undeploy        - Delete resources (keeps namespace)"
+	@echo "  logs            - Tail logs"
+	@echo "  status          - Show objects/status"
+
+.PHONY: build
+build:
```

checkmake (0.2.2)

[warning] Target body for "build" exceeds allowed length of 5 (10).

(maxbodylength)